metadata/co.jp.json

```diff
@@ -1,4 +0,0 @@
-{
-	"domain": "co.jp",
```

Comment:
It looks like this is removing `"domain": "co.jp",` from the file?

Reply:
No metadata, file deleted.

Reply:
👍